scripts/spack/packages/axom/package.py

```diff
@@ -297,9 +299,9 @@ def initconfig_hardware_entries(self):
             # These flags are already part of the wrapped compilers on TOSS4 systems
             #hip_link_flags = "-Wl,--disable-new-dtags -L{0}/lib -L{0}/../lib64 -L{0}/../lib -Wl,-rpath,{0}/lib:{0}/../lib:{0}/../lib64 -lamdhip64 -lhsakmt -lhsa-runtime64".format(hip_root)
 
-            if self.compiler.fc is not None:
+            if "+fortran":
```

Comment:
`if "+fortran" in spec` ?

Reply:
i mean if you want to be all picky and correct....

Reply:
Fixed! It's like it was never wrong.  Thanks @ltaylor16 !